index.bs

```diff
@@ -909,58 +908,75 @@ partial interface HTMLIFrameElement {
       policy <var>inherited policy</var> and declared policy <var>declared
       policy</var>.
       </li>
+      <li>Return <var>policy</var>.</li>
+    </ol>
+  </section>
+  <section>
+    <h3 id="algo-initialize-for-document"><dfn export
```

Comment:
This can also eventually be removed, I hope.

Reply:
Yes.. That is the desire. A 3 phase patch...
1) Make algorithms available in this spec
2) Change HTML spec to call them
3) Remove dead algorithms in spec